build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+import com.android.build.gradle.api.AndroidBasePlugin
+
+plugins {
+    id("com.android.application") version "8.4.1" apply false
+    id("com.android.library") version "8.4.1" apply false
+    id("org.jetbrains.kotlin.android") version "2.0.0" apply false
+    id("com.vanniktech.maven.publish") version "0.28.0" apply false
+    id("com.diffplug.spotless") version "6.25.0" apply false
+}
+
+subprojects {
+    plugins.withType<com.android.build.gradle.api.AndroidBasePlugin>().configureEach {
+        android {
```

Comment:
Did you test this on your local? It's Android extension here, this project can't be built.

Reply:
Yes this part doesn't work
I guess I need to migrate this part to extension

Reply:
You can copy what you need from https://github.com/LawnchairLauncher/lawnicons

Reply:
It's almost done, just need to do a minor changes